src/ui/QuerySummary/QuerySummary.ts

```diff
@@ -260,22 +261,28 @@ export class QuerySummary extends Component {
       l('CancelLastAction')
     );
 
-    cancelLastAction.on('click', () => {
-      this.usageAnalytics.logCustomEvent<IAnalyticsNoMeta>(analyticsActionCauseList.noResultsBack, {}, this.root);
-      this.usageAnalytics.logSearchEvent<IAnalyticsNoMeta>(analyticsActionCauseList.noResultsBack, {});
-      if (this.lastKnownGoodState) {
-        this.queryStateModel.reset();
-        this.queryStateModel.setMultiple(this.lastKnownGoodState);
-        $$(this.root).trigger(QuerySummaryEvents.cancelLastAction);
-        this.queryController.executeQuery();
-      } else {
-        history.back();
-      }
-    });
+    new AccessibleButton()
+      .withElement(cancelLastAction)
+      .withClickAction(() => this.cancelLastAction())
```

Comment:
`withSelectAction` takes care of both click and enter 😄

Reply:
Woops, I'll fix that. I'll check more carefully from now on.